app/components/cases/cases-filter/template.hbs

```diff
@@ -1,8 +1,24 @@
 <div class="vl-form-grid vl-form-grid--is-stacked vl-u-flex-align-right vl-u-spacer-extended-bottom-s">
-  <div class="vl-col--4-12 vl-col--1-1--s">
-    <label class="vl-form__label" for="dossierId">
-      {{t "agendaitem-case"}}
-    </label>
+  <div class="vlc-input-field-block vl-col--4-12">
+    <div class="vl-u-display-flex">
+      <label class="vl-form__label u-padding--right--tinier" for="dossierId">
+        {{t "agendaitem-case"}}
+      </label>
+      <span onclick={{action "initTrigger"}} class="vl-icon vl-icon--small vl-vi vl-vi-info-filled vlc-u-cursor-pointer" aria-hidden="true">
```

Comment:
If you do change the naming don't forget to do it here as well. Instead of `initTrigger`, use `openPopover`.